python/langsmith/client.py

```diff
@@ -3713,11 +3715,26 @@ def list_examples(
             params["dataset"] = dataset_id
         else:
             pass
+        if include_attachments:
+            params["select"] = ["attachment_urls", "outputs", "metadata"]
         for i, example in enumerate(
             self._get_paginated_list("/examples", params=params)
         ):
+            attachment_urls = {}
+            if example["attachment_urls"]:
+                for key, value in example["attachment_urls"].items():
+                    response = requests.get(value["presigned_url"], stream=True)
```

Comment:
Should we do this in parallel under a semaphore or something? Seems like a lot of data to sequentially wait for